addons/crm/crm_lead_view.xml

```diff
@@ -707,8 +702,18 @@
             <field name="ref_ir_act_window" ref="crm.action_lead_mass_mail"/>
         </record>
 
+        <record id="action_mark_as_archive" model="ir.actions.server">
```

Comment:
why ? useless. not required. we have already server action for "Mark As Lost" . and for "archive" we just add "active" field in lead list view.